omnigibson/configs/fetch_behavior.yaml

```diff
@@ -61,8 +59,11 @@ robots:
     controller_config:
       base:
         name: DifferentialDriveController
+      trunk:
+        name: JointController
       arm_0:
         name: InverseKinematicsController
+        dependencies: [trunk]
```

Comment:
Can we change this to `subsume_controllers:` to make it very very explicit what's going on? Also in the code that parses these.

Reply:
done